ext/GeneralizedGeneratedExt.jl

```diff
@@ -0,0 +1,9 @@
+module GeneralizedGeneratedExt
+
+using GeneralizedGenerated, DiffEqBase
```

Comment:
The reason this is failing is that for requres compatability you need
```suggestion
using DiffEqBase
isdefined(Base, :get_extension) ? (using GeneralizedGenerated) : (using ..GeneralizedGenerated)
```
as described in https://pkgdocs.julialang.org/dev/creating-packages/#Requires.jl (because if `GeneralizedGenerated` is loaded by Requires then it will be in the parent rather than the extension.

Reply:
(the same applies to all of these)